Makefile

```diff
@@ -66,8 +66,10 @@ install: libdragon.a libdragonsys.a
 	install -m 0644 include/debug.h $(INSTALLDIR)/mips64-elf/include/debug.h
 	install -m 0644 include/usb.h $(INSTALLDIR)/mips64-elf/include/usb.h
 	install -m 0644 include/console.h $(INSTALLDIR)/mips64-elf/include/console.h
+	install -m 0644 include/joybus.h $(INSTALLDIR)/mips64-elf/include/joybus.h
```

Comment:
Can we hold on installing this (and making `joybus_exec` public API)? I think the API might change as part of the integration with the kernel, so if we can hold up a little on this, we can avoid later breaking things that start depending on it.

We can keep the include file internal, so that only libdragon itself can use `joybus_exec` for now.

Reply:
I'm on the fence about this. On the one hand, I agree that it's probably a good idea not to expose APIs that are subject to change in the near future, but I also see immense utility in exposing the Joybus commands outside of libdragon (for example, this Randnet Keyboard test ROM: https://github.com/meeq/libdragon/commit/44aff08151a3e8d667e14dff59c806f7acc4ae30)

I think there's going to be a huge shake-up in many of the APIs once the kernel lands, so I think we need to have a clear plan for the post-kernel transition process that either minimizes breakage or draws hard lines on where breakage is okay.